scipy/stats/tests/test_multivariate.py

```diff
@@ -1484,8 +1487,11 @@ def test_pairwise_distances(self):
         np.random.seed(514)
 
         def random_ortho(dim):
-            u, _s, v = np.linalg.svd(np.random.normal(size=(dim, dim)))
-            return np.dot(u, v)
+            M = np.random.randn(dim, dim)
```

Comment:
Same as above

Reply:
https://github.com/scipy/scipy/pull/14411#discussion_r670258654

Reply:
For what is worth, here the returned matrix is indeed distributed as the Haar measure, but I figured that I would change it for the more standard and faster implementation via the QR decomposition. I can revert the change if you want.